rotkehlchen/tests/unit/decoders/test_weth.py

```diff
@@ -465,3 +465,117 @@ def test_wxdai_wrap(database, gnosis_inquirer, gnosis_accounts):
     ]
 
     assert events == expected_events
+
+
+# Only checking number of events, the events itself are tested
+# in the test_weth_deposit and test_weth_withdraw
+@pytest.mark.vcr()
+@pytest.mark.parametrize('arbitrum_one_accounts', [['0xBE6660FBE96B61B72Bf35FFaB40eB2CA886A7f85']])
+def test_weth_withdraw_arbitrum_one(database, arbitrum_one_inquirer):
+    evmhash = deserialize_evm_tx_hash(
+        '0xc19c7e1e0af7819b1922a287d034540e8f8dba4e065317d6483d48ac27e727e9',
+    )
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=arbitrum_one_inquirer,
+        database=database,
+        tx_hash=evmhash,
+    )
+    assert len(events) == 3
```

Comment:
I would check as we do in the other tests that all the 3 events are the ones that you expect. It could be that it didn't decode properly some of the events